src/crush/CrushWrapper.h

```diff
@@ -1094,6 +1094,34 @@ class CrushWrapper {
     for (int i=0; i<numrep; i++)
       out[i] = rawout[i];
   }
+  
+  int check_crush_rule(int ruleset, int type, int size, string &error) {
```

Comment:
@tchaikov  En,Here may not use ostream&  because someplace use stringstream.May be after we change all the output to ostream,then we change it here.